tests/base/boring_model.py

```diff
@@ -75,7 +75,12 @@ def training_step(...):
 
         """
         super().__init__()
-        self.layer = torch.nn.Linear(32, 2)
+
+        self.batch_size = 1
+        self.learning_rate = 0.1
+        self.optimizer_name = "SGD"
+        self.layer = torch.nn.Linear(in_features, out_features)
+        self.save_hyperparameters()
```

Comment:
let's move these parameters to init args, as now they won't be saved with `save_hyperparameters`

Reply:
Sure! I moved parameters to __init__ :100: